css-ui-4/Overview.bs

```diff
@@ -2166,6 +2166,129 @@ Document languages should specify
 for each <a>widget</a> that they define
 what needs to be preserved.
 
+The possible kinds of widgets are ''appearance/none'', ''button'', ''searchfield'', ''textfield'',
```

Comment:
These are not CSS property values, and so shouldn't use `''foo''` syntax. They should be defined terms. Something like this:

```
The possible <dfn lt="kind of widget|kinds of widgets">kinds of widgets</dfn> are <dfn for="kind of widget" export>none</dfn>, ...
```

The usages of those terms would then be `[=kind of widget/none=]`

Reply:
I see, got it!